Source/Immutable/Public/Immutable/ApplicationConfig.h

```diff
@@ -0,0 +1,162 @@
+﻿#pragma once
+
+#include "ApplicationConfig.generated.h"
+
+/**
+ * @class UApplicationConfig
+ * @brief Configuration settings for Passport and various APIs.
+ * @details This class stores configuration settings such as URLs, chain names, contract addresses, 
+ * client IDs, and environment settings for the zkEVM API, Orderbook API, and Passport.
+ */
+UCLASS(Abstract, Blueprintable, ClassGroup = Immutable)
+class UApplicationConfig : public UObject
```

Comment:
Can you make this blueprint friendly ? (`BlueprintPure` or `BlueprintReadOnly`)

Reply:
This UCLASS not UFUNCTION

Reply:
Unless, there is another meaning in your comment?

Reply:
Sorry, what I meant was, the getters of this `UCLASS` use `BlueprintPure` or member variables of this `UCLASS` use `BlueprintReadOnly`

You can use either one

Reply:
Currently, the new Passport Initialise method (with a reduced method signature) does not have its own blueprint node, as I don’t think we need to expose it for now. I am using it in the sample game directly from the source code. This is one of several steps toward creating better utilities. As for the other configuration API data, these APIs can only be used from the source code, so there’s no need to expose them to blueprints